src/previewSupport/handleConsoleLogs.js

```diff
@@ -0,0 +1,68 @@
+import isString from 'lodash/isString';
+import channel from './channel';
+// eslint-disable-next-line no-eval
+const globalEval = window.eval;
+
+function reducer(accumulator, currentValue) {
+  let evaluatedValue;
+  if (isString(currentValue)) {
+    evaluatedValue = currentValue;
+  } else {
+    evaluatedValue = globalEval(currentValue);
+  }
+
+  return `${accumulator} ${JSON.stringify(evaluatedValue)}`;
+}
+
+function notifyChannel(args) {
+  if (args.length > 0) {
+    const output = args.reduce(reducer);
+    channel.notify({
+      method: 'log',
+      params: {output},
+    });
+  }
+}
+
+export default function handleConsoleLogs() {
+  /* eslint-disable no-console */
+  const browserConsoleLog = console.log.bind(console);
+  const browserConsoleDebug = console.debug.bind(console);
```

Comment:
Let’s make sure `console.debug` exists in the oldest browser versions we support ([according to caniuse the other ones do](https://caniuse.com/#search=console))